### PR TITLE
Tweak closed cards styling

### PR DESCRIPTION
**Pull request by owennw**

### CODE REVIEW

File: src/closedWindows/closedCard.less [hunk 1]
```diff
@@ -1,22 +1,25 @@
...
 .closed-stock {
-  font-size: 13px;
+  font-size: 12px;
+  font-weight: 400;
 }
 
 .closed-card {
-  height: 60px;
-  border-bottom: 1px solid @separator-colour;
+  height: 70px;
```
Comment (bjedrzejewski) [inline src/closedWindows/closedCard.less]: If you have 5 closed tabs, this won't work.

Comment (bjedrzejewski) [inline src/closedWindows/closedCard.less]: image

Comment (owennw) [inline src/closedWindows/closedCard.less]: Yes I was thinking that the window would auto-resize and we just hadn't done that yet, but I notice before it allowed 5 so I'll push a commit to keep that with the new sizes.

**Comment by owennw** [comment]
You could've just merged it outright, there's no need to merge dev back into this first.

**Pull request merged**